lblod/frontend-lokaal-mandatenbeheer
# Lmb 414 | split group mandates in person table

## Description

We do not want to show the mandates of a person as a comma separated link string. Make it possible to show a dropdown of the rows for the mandaat with the orgaan.

## How to test

Go to mandatarissen module and click open a persoon. The mandaat and orgaan should be clickable and bring you to the correct place.
